apps/transport/lib/jobs/clean_on_demand_validation_job.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Transport.Jobs.CleanOnDemandValidationJob do
+  @moduledoc """
+  This job is in charge of archiving `DB.MultiValidation` after a given period.
```

Comment:
Suggestion (si j'ai bien compris le scope):

```suggestion
  This job is in charge of archiving on-demand-related `DB.MultiValidation` records after a given period.
```